src/dlr.cc

```diff
@@ -184,14 +184,12 @@ void DLRModel::SetupTVMModule(const std::string& model_path) {
 void DLRModel::SetupTreeliteModule(const std::string& model_path) {
   ModelPath paths = get_treelite_paths(model_path);
   int num_worker_threads = -1; // use the maximum amount of threads
-  int include_master_thread = 1;
```

Comment:
What is it for?

Reply:
This corresponds to the API change in the upstream. The `include_master_thread` parameter was causing confusion and thus removed in dmlc/treelite#79